test/jdk/java/awt/print/PrinterJob/PrinterDevice.java

```diff
@@ -23,22 +23,33 @@
 
 /*
  *
- * @bug 4276227
+ * @bug 4276227 8320443
```

Comment:
There's no `@test` tag, the test will not run without it.

Is it intentional?

Reply:
@test is added